.gitignore

```diff
@@ -5,3 +5,4 @@
 
 node_modules
 coverage
+package-lock.json
```

Comment:
Ah. npm gives the reasons to, is there anywhere I can find the reason you wouldn't want to? Alternatively you can always just remove this from the PR so I can merge it...